CriticalMass/TweetTableViewCell.swift

```diff
@@ -68,6 +68,12 @@ class TweetTableViewCell: UITableViewCell, MessageConfigurable, IBConstructable
         handleLabel.text = "@\(tweet.user.screen_name)"
         userNameLabel.text = tweet.user.name
         userImageView.sd_setImage(with: URL(string: tweet.user.profile_image_url_https), placeholderImage: UIImage(named: "Avatar"))
+        if #available(iOS 13.0, *) {
+            // workaround for iOS 13
+            // set background color to clear to keep the same color for background text view and background cell view
+            // when the cell is highlighted
```

Comment:
is the textview backgroundcolor on pre iOS 13 clear by default?

Reply:
I set `Default` color in storyboard. It looks like it `white` in pre iOS 13 (`UIExtendedGrayColorSpace 1 1`). 
But if I set `white`,  it stays `white` in iOS 13 when cell selected:

![Simulator Screen Shot - iPhone 8 Plus - 2019-10-17 at 16 08 22](https://user-images.githubusercontent.com/2142832/67016433-79f27800-f0f8-11e9-88bf-c002c7110a18.png)